security/medusa/l2/kobject_socket.c

```diff
@@ -52,7 +51,6 @@ int socket_kern2kobj(struct socket_kobject * sock_kobj, struct socket * sock)
 		}
 	}
 	sock_kobj->med_object = sk_sec->med_object;
-	return MED_YES;
```

Comment:
leave return code as was

Reply:
If it should have a return code, I suggest to use Kernel codes and not Medusa codes

Reply:
yes, I'll make this change as soon as possible